src/libraries/System.Net.Http/tests/StressTests/HttpStress/build-local.sh

```diff
@@ -1,67 +1,24 @@
 #!/usr/bin/env bash
 
-## This is a helper script for non-containerized local build and test execution.
-## It downloads and uses the daily SDK which contains the compatible AspNetCore bits.
-## Usage:
-## ./build-local.sh [StressConfiguration] [LibrariesConfiguration]
+# Stop script if unbound variable found (use ${var:-} if intentional)
+set -u
 
-version=9.0
-repo_root=$(git rev-parse --show-toplevel)
-daily_dotnet_root=./.dotnet-daily
+# Stop script if command returns non-zero exit code.
+# Prevents hidden errors caused by missing error code propagation.
+set -e
 
-stress_configuration="Release"
-if [ "$1" != "" ]; then
-    stress_configuration=${1,,}                   # Lowercase all characters in $1
-    stress_configuration=${stress_configuration^} # Uppercase first character
-fi
+source="${BASH_SOURCE[0]}"
 
-libraries_configuration="Release"
-if [ "$2" != "" ]; then
-    libraries_configuration=${2,,}                      # Lowercase all characters in $1
-    libraries_configuration=${libraries_configuration^} # Uppercase first character
-fi
+# resolve $source until the file is no longer a symlink
```

Comment:
Why is this needed?

Reply:
It was present in the original scripts, I don't know what the original problem it was solving was.